packages/phpstan-rules/tests/Rules/PreferredMethodCallOverIdenticalCompareRule/Fixture/DependencyInjectionRector.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\PHPStanRules\Tests\Rules\PreferredMethodCallOverIdenticalCompareRule\Fixture;
+
+use PhpParser\Node;
+
+final class DependencyInjectionRector
+{
+    private $rector;
+
+    public function __construct(AbstractRector $rector)
+    {
+        $this->rector = $rector;
+    }
+
+    public function refactor(Node $node): ?Node
+    {
+        $this->rector->getName($node) === 'hey';
```

Comment:
Nice work :+1: 

The opposite should we covered as well:

```php
$this->rector->getName($node) !== 'hey';
```

Reply:
implemented.